client/src/components/Register/Introduction.js

```diff
@@ -0,0 +1,100 @@
+/* global window */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const fallbacks = require('../../../lang/src/en.json');
+
+function Introduction({ canSkip, onContinue, onSkip }) {
+  const { ss: { i18n } } = window;
+
+  return (
+    <div>
+      <h2 className="mfa-section-title">
+        { i18n._t('MFAIntroduction.TITLE', fallbacks['MFAIntroduction.TITLE']) }
+      </h2>
+
+      <h4 className="mfa-feature-list-title">
+        { i18n._t('MFAIntroduction.HOW_IT_WORKS', fallbacks['MFAIntroduction.HOW_IT_WORKS']) }
+      </h4>
+
+      <ul className="mfa-feature-list">
+        <li className="mfa-feature-list-item">
+          <img
+            alt="Shields indicating additional protection"
+            aria-hidden="true"
+            className="mfa-feature-list-item__icon"
+            src="/resources/vendor/silverstripe/mfa/client/dist/images/extra-protection.svg"
```

Comment:
We'll need to inject this as a prop. Guy's added the URL as part of the data that the start() method returns, not sure if the introduction has one but I'd guess it's  `indexAction()` or something?

Reply:
This probably makes sense to be returned as part of the schema?

Reply:
Hmm, they'll have to come down via the base schema endpoint I think, otherwise there's two roundtrips required to render the UI for the first time. I'll sort it.

Reply:
You could provide the MFA module path from the schema then reference the assets relatively from the frontend, maybe